Antonio_Ramos_Documentation

```diff
@@ -0,0 +1,233 @@
+
+- evaluated attribute tables to ensure that the proper data column names in the table were being referenced by the K-function Local_3Stage.py program. All data table names are correctly referenced, however, there is also another attribute "geometry" which is being listed on lines 39-41 and I am not sure what that is doing, it does not appear in any of the attribute tables from the data files. 
+
+- when running line by line analysis of K-function Local_3Stage.py I again get the infinite simulation loops when running lines 104-137 together. Upon further review I did not get any errors running lines 104-119 together, but it does not get past 50 percent complete on the simluations, have to interupt program with "control C". Could be because I am running the lines independent of any data. Now, when I run lines 120-128 together I get "SIMULATION 1 Threw an error". Moreover, when I run line 122 in K-fucntion Local_3Stage.py, I get the following error...
+  "File "G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\kfunctions\Three Stage K-functions\kfunction_3Stage.py", line 49, in countSimulation
+    fsFeatures['FS_PROBS'] = fsFeatures[[fieldFS]] / float(sum(fsFeatures[fieldFS]))
+
+TypeError: unsupported operand type(s) for +: 'int' and 'str'"
+
+-(now working in kfunction_3Stage.py script) when running lines 42-49 (so as to include the definition of "fsFeatures") no error occurs and script runs perfectly fine. Moreover, when running the entire function definied as "countSimulation" (lines 42-87) no error is retunred and function executes. Similarly when running the entire function defined as "pointSimulation" (which is referenced in line 122 of K-function Local_3Stage.py) no error is returned and function executes. So I am confused as to what is sending us into this infinite loop. I believe it might have to do with the data we are feeding into the programs but I am not sure why. As mentioned above the attribute names being referenced coincide between the program and the data. 
+
+- Will mentions having come across a similar problem in his documentation on lines 143-145. I sent him an email asking if he remembers how he got around that problem, his solution is not articulated in the documentation. 
+
+06/11/2020
+- after communicating with Will, we were still not able to get documentation which aligns with the data referenced in the python scripts. Documentation that Will sent over was the same as what was already in the github repository. File not downloaded since it is the same as what is in the git repo. 
+
+- finished filling in Data & Script File Table which contains a list of all the files mentioned by Will Heikoff in his documentation, or utilized by the python script "K-function Local_3Stage.py" saved it in G:\MAX-Filler\Collab\Labs-kbuzard-S18\Admin . 
```

Comment:
As I was reading through this to try to make sense of everything, I realized one more column would be helpful: file type (as in Python script, R script, etc.). Would you please add this when you have a chance?

Reply:
Will do.